lib/Runtime/ByteCode/ByteCodeGenerator.cpp

```diff
@@ -4377,36 +4385,30 @@ void Bind(ParseNode *pnode, ByteCodeGenerator *byteCodeGenerator)
             }
         }
 
-        if (pnode->AsParseNodePid()->sym)
+        if (pnodeName->sym)
         {
-            pnode->AsParseNodePid()->sym->SetIsUsed(true);
+            pnodeName->sym->SetIsUsed(true);
         }
 
         break;
     }
     case knopMember:
     case knopMemberShort:
     case knopObjectPatternMember:
-        if (pnode->AsParseNodeBin()->pnode1->nop == knopComputedName)
```

Comment:
Don't see this in below conditional. Did we remove this case?

Reply:
It is redundant.  If it is knopComputedName, it will skip the if for knopStr and break anyways.

---
In reply to: [180852627](https://github.com/Microsoft/ChakraCore/pull/4969#discussion_r180852627) [](ancestors = 180852627)